src/App.css

```diff
@@ -0,0 +1,39 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.App-header {
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
+
```

Comment:
- [ ] Nice jobe so far, but as the requirements say, we should stick to the original design. Kindly make sure the white backgound is not visible when your conntent is being filtered, keep the baground of the template.

<img width="292" alt="image" src="https://github.com/Elhambasir/weather/assets/65068771/47f4480d-0dfd-4d32-8c8b-bc438d004e28">

Reply:
Ok thank you